src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -487,7 +506,7 @@ See the accompanying LICENSE file for applicable license.
           filedirparameter="FILEDIR">
         <ditaFileset format="dita" processingRole="normal"/>
         <param name="TRANSTYPE" expression="${transtype}"/>
-        <param name="FILTERFILEURL" expression="${dita.input.filterfile.url}"/>
+        <param name="FILTERFILEURL" expression="${mergedditaval.url}"/>
```

Comment:
Sure - that makes sense, and we can keep using the old one. The reason I changed it is that previously we didn't create a URL property for this until `flag-module`  -- now that I'm creating one earlier on, I reused the early version. I'm not sure whether it's best to create it early as `dita.input.filterfile.url` or to use two different properties for the two different cases.